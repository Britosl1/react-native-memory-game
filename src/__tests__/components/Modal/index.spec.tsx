import React from 'react';
import {render} from '@testing-library/react-native';
import VictoryModal from '../../../components/Modal';

describe('Snapshot test', () => {
  test('Modal component', () => {
    const {toJSON} = render(
      <VictoryModal
        onPress={jest.fn()}
        onPressHome={jest.fn()}
        rounds={2}
        visible={true}
      />,
    );

    expect(toJSON()).toMatchSnapshot();
  });
});
