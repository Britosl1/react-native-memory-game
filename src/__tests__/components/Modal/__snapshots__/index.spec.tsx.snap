// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test Modal component 1`] = `
<Modal
  animationType="slide"
  hardwareAccelerated={false}
  visible={true}
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#b491c8",
          "height": "100%",
          "justifyContent": "center",
          "paddingBottom": 50,
          "paddingLeft": 50,
          "paddingRight": 50,
          "paddingTop": 50,
        },
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "marginBottom": 20,
          },
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "alignItems": "center",
              "color": "white",
              "fontSize": 20,
              "fontWeight": "800",
            },
          ]
        }
      >
        Victory!
      </Text>
      <Text
        style={
          Array [
            Object {
              "alignItems": "center",
              "color": "white",
              "fontSize": 20,
              "fontWeight": "800",
            },
          ]
        }
      >
        Number of rounds: 2
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={true}
      nativeID="animatedComponent"
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "#663a82",
              "borderRadius": 6,
              "height": 50,
              "justifyContent": "center",
              "marginBottom": 50,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "white",
                "fontWeight": "800",
              },
            ]
          }
        >
          Play Again!
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={true}
      nativeID="animatedComponent"
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "#663a82",
              "borderRadius": 6,
              "height": 50,
              "justifyContent": "center",
              "marginBottom": 50,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "white",
                "fontWeight": "800",
              },
            ]
          }
        >
          Ranking
        </Text>
      </View>
    </View>
  </View>
</Modal>
`;
